### ISSUE TITLE
The Web Client needs a page to edit TSDB properties

### ISSUE THREAD

**Issue by wjonassen** [opendcs/rest_api#6]
The Java OpenDCS client has some functionality for things like default site name preferences.  The web client needs something like this too.

The solution is to have a web interface to edit some properties.  The web interface can display, by default, the properties that affect the GUI, with hovertext explaining how it affects the gui.

Additionally, each specific property will need to be implemented in the client, but the first step is to add this page.  The specific properties can be identified in separate feature requests.

**Comment by jbatterman** [comment]
Agreed this is a need.  In addition, users should be able to maintain different properties for different profiles/office IDs.  For example, I maintain 4 different profiles (NWO T7, NWK T7, HEC T7 NWK, and HEC T7 NWDM).

**Comment by jbatterman** [comment]
Not sure if this should be a separate issue. It would be nice if column widths were able to be resized.  Potentially this could be saved under some user properties page.

**Comment by jbatterman** [comment]
It looks like the Platforms tab is utilizing a variety of different configured site names on the initial load rather than being consistent with one configured site name (i.e., some show the USGS ID, some show the NWS Handbook 5 ID, some show the CWMS ID).  This is typically configured in the profile to have a preference on which site name type is shown, but even without that should probably be consistent by the preference.  In the example below, the red square is using the USGS ID, the blue square is using the handbook 5 ID, and the yellow square is using the CWMS ID.

<img width="661" height="446" alt="Image" src="https://github.com/user-attachments/assets/599a7871-6329-4ce1-816a-048a84e28f26" />
